example-application/data-access/seeders/20191229151823-countries.js

```diff
@@ -1,22 +1,21 @@
 module.exports = {
-  up: (queryInterface, Sequelize) => {
+  up: async (queryInterface, Sequelize) => {
     //Configuration ✅
-    queryInterface.bulkInsert('Countries', [{
+    await queryInterface.bulkInsert('Countries', [{
       name: 'Italy',
       name: 'USA',
       name: 'India'
-    }], {})
+    }], {});
 
     //Test data ❌ 
```

Comment:
Keep the formatting! 

This is how we've done it:

// ❌ Anti-Pattern: No clean-up for the network interceptions, the next test will face the same behaviour

p.s. Put a reference, the name of the bullet, that explains this